kolibri/core/assets/src/vue/error-box/index.vue

```diff
@@ -21,9 +21,9 @@
     $trNameSpace: 'error',
     $trs: {
       errorHeader: 'Error',
-      errorButtonLabel: 'Hide Error',
+      errorButtonLabel: 'Hide error',
       // eslint-disable-next-line
-      explanation: `Sorry, we've encountered an issue. You may need to refresh the page or click one of the main navigation links.`,
+      explanation: `Unknown issue. You may need to refresh`,
```

Comment:
I really, really dislike "Unknown issue" messages... Isn't there any recommended action we can suggest to the user to escalate the issue? 

How does this sound?
`Ups, something went wrong! Please try refreshing the page and if the problem persists, contact your administrator.`